Source/Softswitch/inc/poets_msg.h

```diff
@@ -3,69 +3,113 @@
 
```

Comment:
I've always felt that this file really is in the wrong place - it should be in `Source/Common`, because it's used by just about everything of importance (Root, Mothership, Softswitch, notably excepting Nameserver), and it's very strange to make an exception for this file. Could we reasonably move it? :sweet_potato:

Reply:
potentially, let me have a think.

Reply:
Moved in e0fee08bc0056ec8d3e2bbdc8495efa71c1c6459